src/components/views/SSICredentialViewItem/CredentailViewImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+
+import {IBasicCredentialLocaleBranding} from '@sphereon/ssi-sdk.data-store';
+import {Image} from 'react-native';
+import {
+  SSICredentialViewItemBackgroundImageStyled as BackgroundImage,
+  SSICredentialViewItemCardStyled as Card,
+  SSICredentialViewItemImageContainerStyled as ImageContainer,
+  SSICredentialViewItemLogoContainerStyled as LogoContainer,
+  SSICredentialViewItemLogoImageStyled as LogoImage,
+} from '../../../styles/components';
+
+export const CredentialViewImage = ({branding}: {branding: IBasicCredentialLocaleBranding}) => {
+  const CARD_ASPECT_RATIO = 3 / 2;
+  const backgroundURI = branding.background?.image?.uri;
+  const backgroundColor = branding.background?.color ? branding.background.color : 'white';
+  const logoURI = branding.logo?.uri;
+  const [logoAspectRatio, setLogoAspectRatio] = useState(1);
+
+  if (logoURI) {
+    Image.getSize(logoURI, (width, height) => height && setLogoAspectRatio(width / height));
```

Comment:
Memory leak, it will do this on each render and not clean up the callback when it unmounts.
```suggestion
 useEffect(() => {
    if (logoURI) {
      Image.getSize(
        logoURI,
        (width, height) => height && setLogoAspectRatio(width / height),
        (error) => {
          console.error('Failed to get image size:', error);
          setLogoAspectRatio(1); // Fallback aspect ratio
        }
      );
    }
  }, [logoURI]);
```

Reply:
the useEffect is taking care of calling when it needs to be called only